src/processor/reconnection-graph.service.ts

```diff
@@ -48,6 +50,14 @@ export class ReconnectionGraphService implements OnApplicationBootstrap, OnAppli
     const [graphConnections, graphKeyPair] = await this.getUserGraphFromProvider(dsnpUserId, providerId);
     this.logger.log("graphConnections", graphConnections);
     this.logger.log("graphKeyPair", graphKeyPair);
```

Comment:
I left these log statements in place until the connections could be pulled into graph SDK